multinet/migrations/util.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities for migrations."""
+import inspect
+from datetime import datetime
+from arango.collection import StandardCollection
+from multinet.db import system_db
+from multinet.migrations.base import BaseMigration
+from multinet.migrations.types import Migration, MigrationDocument
+from types import ModuleType
+from typing import List, Generator
+
+
+# TODO: Add creation and return of `migrations` collection in _system db. This will
+# hold a document for each migration that exists. This can be used to check the
+# migrations that have been run, what needs to be run, etc.
```

Comment:
I'd like to never leave a TODO in the code like this if it's not referencing a GitHub issue. Can you either complete this work now, or file the issue?

Reply:
Just something I forgot to remove, this is addressed now. I'll remove it.